test/functional/p2p_sendheaders.py

```diff
@@ -380,6 +380,8 @@ def test_nonnull_locators(self, test_node, inv_node):
         # PART 3.  Headers announcements can stop after large reorg, and resume after
         # getheaders or inv from peer.
         for j in range(2):
+            self.log.info("Part 3: iteration level: {}".format(j))
+            
```

Comment:
I'd slightly prefer if the comments in the loops down there were transformed into debug statements. (Where we check for `i == ...` and `j == ..`